### ISSUE TITLE
Awesomeplete not highlighting matches

### ISSUE THREAD

**Issue by danielabar** [CANVE/canve-viz#47]
Also the search matching seems a little off.

**Comment by matanox** [comment]
Might be a newer release of awesomeplete by now...

**Comment by danielabar** [comment]
Absolutely, might as well upgrade to latest version as part of this fix.
Could also be the css isn't being loaded.
